CO2_Gadget.ino

```diff
@@ -61,6 +61,13 @@ uint16_t measurementInterval = 10;
 bool bleInitialized = false;
 int8_t selectedCO2Sensor = -1;
 bool outputsModeRelay = false;
+bool activeBuzzer = true;
```

Comment:
Should be false by default

Reply:
As this implementation is done without menu configuration, if you set activeBuzzer = false you can't test it.